src/doghouse/adapters/github/gh_cli_adapter.py

```
@@ -0,0 +1,190 @@
+                    if comments:
+                        first_comment = comments[0]
+                        msg = first_comment.get("body", "Unresolved thread")
+                        if len(msg) > 80:
+                            msg = msg[:77] + "..."
+
+                        blockers.append(Blocker(
+                            id=f"thread-{first_comment['id']}",
+                            type=BlockerType.UNRESOLVED_THREAD,
+                            message=msg
+                        ))
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)